bundown/package.json

```diff
@@ -19,13 +20,16 @@
     "micromark-extension-gfm": "^3.0.0",
     "micromark-extension-math": "^3.0.0",
     "string-width": "^7.1.0",
+    "ts-pattern": "^5.0.6",
+    "unist-util-visit": "^5.0.0",
     "wrap-ansi": "^9.0.0"
   },
+  "peerDependencies": {
+    "typescript": "^5.3.3"
+  },
```

Comment:
Why this change?

Reply:
This is the default on `bun init`. Also we don't have any dependency on typescript (neither do consumers of `bundown` as a lib). That's why it isn't in the dependencies array.